cpp/arcticdb/version/python_bindings.cpp

```diff
@@ -225,6 +225,22 @@ void register_bindings(py::module &version, py::exception<arcticdb::ArcticExcept
         return lst;
     };
 
+    auto adapt_write_dfs = [](std::vector<std::variant<VersionedItem, DataError>> && ret) -> py::list {
+        py::list lst;
+        for (auto &res: ret) {
+            util::variant_match(
+                    res,
+                    [&lst] (VersionedItem& res) {
+                        lst.append(res);
+                    },
+                    [&lst] (DataError& data_error) {
+                        lst.append(data_error);
+                    }
+            );
```

Comment:
When all of the visitors are the same, you can write it more concisely:
```
            util::variant_match(
                    elem,
                    [&lst] (auto& elem) {
                        lst.append(elem);
                    }
            );
```

Reply:
Removed the function entirely.